cypress/objects/loginPage.js

```diff
@@ -3,7 +3,7 @@ class LoginPage {
       cy.visit('/')
 }
 getvalidationerror(){
-    return cy.get('Select.not-valid');
+return cy.get('Select.not-valid');
 
 }
 submitlogin(){
```

Comment:
I'd add "button" at the end so that we know what kind of element we're dealing with here, a button in this case :)